pkg/cmd/run.go

```diff
@@ -104,14 +107,14 @@ func RunMicroshift(cfg *config.Config) error {
 	util.Must(m.AddService(sysconfwatch.NewSysConfWatchController(cfg)))
 	util.Must(m.AddService(controllers.NewKubeAPIServer(cfg)))
 	util.Must(m.AddService(controllers.NewKubeScheduler(cfg)))
-	util.Must(m.AddService(controllers.NewKubeControllerManager(cfg)))
+	util.Must(m.AddService(controllers.NewKubeControllerManager(context.Background(), cfg)))
```

Comment:
Should this be the context that everything else is using to know when to exit?

Reply:
Moved the runCtx declaration above the AddServer chunk so it can be passed down to services